HeavyFlavorAnalysis/SpecificDecay/src/BPHPhiToKKBuilder.cc

```diff
@@ -0,0 +1,168 @@
+/*
+ *  See header file for a description of this class.
+ *
+ *  $Date: 2015-07-24 11:29:20 $
+ *  $Revision: 1.1 $
+ *  \author Paolo Ronchese INFN Padova
+ *
+ */
+
+//-----------------------
+// This Class' Header --
+//-----------------------
+#include "HeavyFlavorAnalysis/SpecificDecay/interface/BPHPhiToKKBuilder.h"
+
+//-------------------------------
+// Collaborating Class Headers --
+//-------------------------------
+#include "HeavyFlavorAnalysis/RecoDecay/interface/BPHRecoBuilder.h"
+#include "HeavyFlavorAnalysis/RecoDecay/interface/BPHPlusMinusCandidate.h"
+#include "HeavyFlavorAnalysis/SpecificDecay/interface/BPHParticlePtSelect.h"
+#include "HeavyFlavorAnalysis/SpecificDecay/interface/BPHParticleEtaSelect.h"
+#include "HeavyFlavorAnalysis/SpecificDecay/interface/BPHMassSelect.h"
+#include "HeavyFlavorAnalysis/SpecificDecay/interface/BPHChi2Select.h"
+#include "HeavyFlavorAnalysis/SpecificDecay/interface/BPHParticleMasses.h"
+
+//---------------
+// C++ Headers --
+//---------------
+using namespace std;
+
+//-------------------
+// Initializations --
+//-------------------
+
+
+//----------------
+// Constructors --
+//----------------
+BPHPhiToKKBuilder::BPHPhiToKKBuilder(
+               const edm::EventSetup& es,
+               const BPHRecoBuilder::BPHGenericCollection* kPosCollection,
+               const BPHRecoBuilder::BPHGenericCollection* kNegCollection ):
+  kPosName( "KPos" ),
+  kNegName( "KNeg" ),
+  evSetup( &es ),
+  posCollection( kPosCollection ),
+  negCollection( kNegCollection ) {
+    ptSel = new BPHParticlePtSelect (  0.7 );
+   etaSel = new BPHParticleEtaSelect( 10.0 );
+  massSel = new BPHMassSelect( 0.9995, 1.0395 );
```

Comment:
should this and a few other defaults be set with respect to corresponding defined constants for masses?